client/components/account-status/account-fees/index.js

```diff
@@ -51,11 +51,15 @@ const AccountFees = ( props ) => {
 			{ feeDescription }
 			<ExpirationBar
 				feeData={ currentFee }
-				currencyCode={ currencyCode }
+				currencyCode={
```

Comment:
We are passing `currentFee` into ExpirationBar, can we drop `currencyCode` property and move the logic into `ExpirationBar`?

Reply:
Addressed in 1921d85